core-blocks/image/index.js

```diff
@@ -217,6 +217,16 @@ export const settings = {
 			/>
 		);
 
+		if ( 'left' === align || 'right' === align ) {
```

Comment:
I know it's the style in core and it's unrelated to your patch, but constant-first does my head in… it feels unlike how anyone would speak logically about the conditional. `</rant>`

This looks like it'll need a `deprecated` handler because this changes block markup.